cmd/cli/main.go

```diff
@@ -168,6 +168,34 @@ func UpdateAction(c *cli.Context) error {
 	return nil
 }
 
+// TerminateAction executes the CLI terminate command
+func TerminateAction(c *cli.Context) error {
+	terminate := operations.TerminateJob{}
+
+	jobNameBase := c.String("job-name-base")
+	if len(jobNameBase) != 0 {
+		terminate.JobNameBase = jobNameBase
+	} else {
+		return cli.NewExitError("unspecified flag 'job-name-base'", -1)
+	}
+
+	mode := c.String("mode")
+	if len(mode) > 0 && mode != "cancel" && mode != "stop" {
+		return cli.NewExitError("unknown value for 'mode', only 'cancel' and 'stop' are supported", -1)
+	}
+
+	terminate.Mode = mode
+	err := operator.Terminate(terminate)
```

Comment:
Nitpicking but empty line before 188 and remove the empty line on 189

Reply:
I agree, there is no need for the empty line.